vizro-ai/README.md

```diff
@@ -7,6 +7,7 @@
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/mckinsey/vizro/blob/main/LICENSE.md)
 [![Documentation](https://readthedocs.org/projects/vizro-ai/badge/?version=latest)](https://vizro-ai.readthedocs.io/)
 [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7858/badge)](https://www.bestpractices.dev/projects/7858)
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mckinsey/vizro/blob/docs/add_colab_link/vizro-ai/examples/colab-example-chart.ipynb)
```

Comment:
I like the icon, but I think it is a bit misleading to open the readme in colab? Should we have a section below to say: "Try out live" or do you think it is clear from the icon that this will open an example?

Reply:
Move it to a "try out live" section and structure into a table format, so we can add dashboard link later. Let me know if you think other format work better

Reply:
YEs, it works much better, and is also good for extension indeed!